tools/accuracy_checker/configs/colorization-v2.yml

```diff
@@ -0,0 +1,40 @@
+evaluations:
+  - name: colorization-v2
+    module: custom_evaluators.colorization_v2_evaluator.ColorizationEvaluator
+    module_config:
+        network_info:
+          test:
+            model:   public/colorization-v2/FP32/colorization-v2.xml
+            weights: public/colorization-v2/FP32/colorization-v2.bin
+            color_reconstruction:
+            -   coeff: public/colorization-v2/colorization-v2.npy
```

Comment:
why it should be list? Do you expect that several options with the same name will be presented in config? If no, please replace to dictionary.

Reply:
@eaidova corrected)